### PR TITLE
psm: dont require liberty models or set voltages when running check_power_grid

### PR DESCRIPTION
**Pull request by gadfort**
Changes:
- Avoids the check for voltages when doing connectivity checks as this is not required.

Adds:
- test for functionality of new behavior.

### CODE REVIEW

File: src/psm/test/check_power_grid.tcl [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Check that check_power_grid works without a liberty file
+# or the voltage on the net being set
+source helpers.tcl
+
+read_lef Nangate45/Nangate45.lef
+read_def gcd.def
+check_power_grid -net VDD
```
Comment (maliberty) [inline src/psm/test/check_power_grid.tcl:7]: output diff?

**Review by maliberty — APPROVED** [review]

**Comment by gadfort** [comment]
@maliberty this one will cause a merge conflict with the other one, so I recommend merging this one last and I can resolve the merge once the other one is done.

**Comment by maliberty** [comment]
I merged it but I don't see conflicts.

**Comment by gadfort** [comment]
@maliberty well that's good, but I think I'd like to move the def file into the Nangate45_data to align with the organization in the other PR. I can push that a little later.

**Comment by gadfort** [comment]
@maliberty it should be good to go now

**Review by maliberty — APPROVED** [review]

**Pull request merged**